src/writer.jl

```diff
@@ -52,6 +52,14 @@ Base.@kwdef struct MarkdownVitepress <: Documenter.Writer
     - `false`: Does not remove the contents of `md_output_path` after the Vitepress site is built.
     """
     clean_md_output::Union{Nothing, Bool} = nothing
+    """
+    DeployDecision from Documenter.jl. This is used to determine whether to deploy the documentation or not.
+    Options are:
+    - `nothing`: **Default**. Automatically determine whether to deploy the documentation.
+    - `Documenter.DeployDecision`: Override the automatic decision and deploy based on the passed config.
+    It might be useful to use the latter if DocumenterVitepress fails to deploy automatically.
```

Comment:
```suggestion
    It might be useful to use the latter if DocumenterVitepress fails to deploy automatically.
    You can pass a manually constructed `Documenter.DeployDecision` struct, or the output of 
    `Documenter.deploy_folder(Documenter.auto_detect_deploy_system(); repo, devbranch, devurl, push_preview)`.